### PR TITLE
Defined "offscreen document"

### PR DESCRIPTION
**Pull request by IanStanion-google**
Added a definition for the term "offscreen document" in the reference page.

### CODE REVIEW

File: site/en/docs/extensions/reference/offscreen/index.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ api: offscreen
 
 ## Manifest
 
+Offscreen documents are invisible webpages with their own DOM that can be created and managed using the `offscreen` API. Requesting this permission allows the extension to use DOM APIs without obtrusively opening new windows or tabs that interrupt the user experience.
```
Comment (oliverdunk) [inline site/en/docs/extensions/reference/offscreen/index.md:7]: Thoughts on moving this above the manifest header?

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Left one comment, otherwise LGTM 👍

File: site/en/docs/extensions/reference/offscreen/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/docs/extensions/reference/offscreen/index.md:7]: I agree. This is a basic offscreen definition. the place for it is here: https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/api/offscreen.idl;l=5

Comment (jpmedley) [reply site/en/docs/extensions/reference/offscreen/index.md:7]: @IanStanion-google, as requested, here's the Gerrit PR: https://chromium-review.git.corp.google.com/c/chromium/src/+/4440738

It doesn't include your second sentence. I think it's more appropriate here.

**Review by jpmedley — APPROVED** [review]

**Comment by oliverdunk** [comment]
@IanStanion-google I think we can merge whenever :) Two things beforehand:

- Could we move this above the header still? e.g in this PR move your line above the line above it?
- We probably don't want to duplicate Joe's text, but I think the first line you had is still useful to have here too if you can reword it somehow.

**Review by oliverdunk — APPROVED** [review]

**Pull request merged**